subsystem/src/manager.rs

```diff
@@ -84,8 +84,11 @@ pub struct Manager {
     shutting_down_tx: mpsc::UnboundedSender<()>,
     shutting_down_rx: mpsc::UnboundedReceiver<()>,
 
-    // List of subsystem tasks.
+    // List of subsystem tasks
     subsystems: Vec<SubsystemInfo>,
+
+    // Stored handle to the lock file
+    lock_file: Option<File>,
```

Comment:
I think this shouldn't be part of subsystems... but should be part of initialization.

Reply:
OK, updated.